packages/components/addon/components/hds/table/index.hbs

```diff
@@ -22,16 +22,22 @@
               {{column.label}}
             </Hds::Table::ThSort>
           {{else}}
-            <Hds::Table::Th @align={{column.align}} @width={{column.width}}>{{column.label}}</Hds::Table::Th>
+            <Hds::Table::Th @align={{column.align}} @width={{column.width}}>
+              {{#if this.isHidden}}
```

Comment:
if the API is going to be

```
@columns={{array
      (hash key="artist" label="Artist" isSortable="true")
      (hash key="album" label="Album" isSortable="true")
      (hash key="year" label="Release Year" isHidden="true")
```
then the only required change would the following line
```suggestion
              {{#if column.isHidden}}
```

Reply:
Ahh yes that does it! 
<img width="1060" alt="CleanShot 2023-07-31 at 10 26 54@2x" src="https://github.com/hashicorp/design-system/assets/4587451/aae804bd-0343-4253-ae9e-8ea6f780a8bc">